sam/models/spc_model.py

```diff
@@ -115,12 +117,13 @@ class SPCRegressor(BaseTimeseriesRegressor):
 
     Parameters
     ----------
-    predict_ahead: integer, optional (default=(1,))
-        how many steps to predict ahead. For example, if (1, 2), the model will predict both 1 and
-        2 timesteps into the future. If (0), predict the present. If not equal to (0),
-        predict the future, with differencing.
+    predict_ahead: integer, optional (default=[0]])
+        how many steps to predict ahead. For example, if [1, 2], the model will predict both 1 and
+        2 timesteps into the future. If [0], predict the present. If not equal to [0],
+        predict the future. Combine with `use_diff_of_y` to get a persistence benchmark forecasting
+        model.
         A single integer is also allowed, in which case the value is converted to a singleton list.
-    quantiles: array-like, optional (default=())
+    quantiles: array-like, optional (default=[])
```

Comment:
Why did we switch back to lists everywhere from tuples?

Reply:
Some assertions/assumptions were still failing for tuples. (for example making a list of a tuple)

I think the most important reason to pick tuples is so we can use `a,b,c = (1,2,3,)` kind of syntax. Which is impossible, since quantiles can have any length that the user wants...

Reply:
I'm okay with using lists, but not as default parameters, since there is always the risk of the mutable default argument, in that case I would use None as default value, or require a value and not give a default one